detox/jest.detox.startup.js

```diff
@@ -2,11 +2,31 @@ const { by, cleanup, init, device, element, waitFor } = require("detox");
 const adapter = require("detox/runners/jest/adapter");
 const specReporter = require("detox/runners/jest/specReporter");
 const config = require("./detox.config");
+const { toMatchImageSnapshot } = require("jest-image-snapshot");
+const path = require("path");
```

Comment:
I think this line can be removed, since you destructure functions off of it below, right?

Reply:
Yes, you're right, we don't need it anymore. I've removed it, thanks 👍